client/a8c-for-agencies/sections/sites/sites-dashboard/sites-dataviews-style.scss

```diff
@@ -21,7 +21,8 @@
 	font-size: rem(14px);
 
 	.status-badge {
-		border-radius: 2px;
+		/* stylelint-disable-next-line scales/radii */
+		border-radius: 12px;
```

Comment:
This is a "sneak-in" change - just to address https://github.com/Automattic/dotcom-forge/issues/8763 - so we don't have to create a separate PR for it.